backend-build.js

```diff
@@ -9,7 +9,7 @@ const { exec } = require("child_process")
 const MASTER_PATH = path.join(__dirname, "amplify", "backend")
 
 
-const API_NAMES = ["AddItem", "BorrowBatch", "BorrowItem","CreateBatch","DeleteBatch", "DeleteItem", "ReturnBatch", "ReturnItem","UpdateDescription","UpdateTags"]
+const API_NAMES = ["AddItem", "BorrowBatch", "BorrowItem","CreateBatch","DeleteBatch", "DeleteItem", "ReturnBatch", "ReturnItem","UpdateDescription","UpdateItemOwner","UpdateTags"]
```

Comment:
Ok for this one, but for the last one, can you split this up into different lines, like this:
```
const API_NAMES = [
    "AddItem",
    "BorrowBatch",
    ....
]
```

It's just getting messing with it being a big long line.